native/selected-text-reader/src/macos.rs

```diff
@@ -22,9 +16,12 @@ type CGEventRef = *mut __CGEvent;
 type CGKeyCode = u16;
 type CGEventFlags = u64;
 const CG_EVENT_FLAG_MASK_COMMAND: CGEventFlags = 0x100000;
+const CG_EVENT_FLAG_MASK_SHIFT: CGEventFlags = 0x020000;
+const CG_EVENT_FLAG_MASK_CONTROL: CGEventFlags = 0x040000;
 
 type CGEventTapLocation = u32;
 const CG_HID_EVENT_TAP: CGEventTapLocation = 0;
+const CG_SESSION_EVENT_TAP: CGEventTapLocation = 1;
```

Comment:
This is important -- session event taps don't get picked up by our hot key listener, which allows us to do keyboard commands while the user has their hot key pressed.